packages/e2e-tests/next-app/pages/ssg-page.tsx

```diff
@@ -1,22 +1,29 @@
 import React from "react";
-import { NextPageContext } from "next";
+import { GetStaticPropsContext } from "next";
 
 type SSGPageProps = {
   name: string;
+  preview: boolean;
 };
 
 export default function SSGPage(props: any): JSX.Element {
   return (
     <React.Fragment>
       {`Hello ${props.name}! This is an SSG Page using getStaticProps().`}
+      <div>
+        <p cy-data="preview-mode">{String(props.preview)}</p>
+      </div>
     </React.Fragment>
   );
 }
 
 export async function getStaticProps(
-  ctx: NextPageContext
+  ctx: GetStaticPropsContext
 ): Promise<{ props: SSGPageProps }> {
   return {
-    props: { name: "serverless-next.js" }
+    props: {
+      name: "serverless-next.js",
+      preview: !!ctx.preview || false
```

Comment:
I sometimes get confused on this, but I don't think `|| false` is needed since you are already adding `!!`?

Reply:
🤦‍♂️ 

of course, undefined is definitely coerced to `false` here. The or guard is not needed for sure 👍